<launch>
	<arg name="width" default="640" />
	<arg name="height" default="480" />
	<arg name="color_file" default="" />
	<arg name="motion_type" default="1" />
	<arg name="rotation_type" default="0" />
	<arg name="number_of_repetitions" default="10" />
	<arg name="time_limit" default="0" />
	<arg name="limb" default="0" />
	<arg name="calibration_duration" default="10" />
	<arg name="robot_position" default="0" />
	<arg name="calibration_file" default="" />
	<arg name="calibration_output_file" default="" />
	
	<node pkg="sound_play" type="soundplay_node.py" name="sound_play" />
	<node pkg="wm_voice_generator" type="wm_voice_component_short.py" name="wm_voice_generator" />
	<node pkg="reha_game" type="Exercises.py" name="reha_game" args="--width $(arg width) --height $(arg height) --color_file $(arg color_file) --motion_type $(arg motion_type) --rotation_type $(arg rotation_type) --number_of_repetitions $(arg number_of_repetitions) --time_limit $(arg time_limit) --limb $(arg limb) --calibration_duration $(arg calibration_duration) --robot_position $(arg robot_position) --calibration_output_file $(arg calibration_output_file) --calibration_file $(arg calibration_file)" output="screen" />
</launch>
